automation/taskcluster/lib/tasks.py

```diff
@@ -163,18 +166,18 @@ def craft_test_task(self, variant):
             },
         )
 
-   
+
     def craft_ui_tests_task(self):
         artifacts = {
             "public": {
                 "type": "directory",
-                "path": "/build/fenix/results",
+                "path": "/build/{}/results".format(_PROJECT_NAME),
                 "expires": taskcluster.stringDate(taskcluster.fromNow(DEFAULT_EXPIRES_IN))
             }
         }
 
         env_vars = {
-            "GOOGLE_PROJECT": "moz-fenix",
+            "GOOGLE_PROJECT": GOOGLE_PROJECT,
```

Comment:
Can you elaborate why this value is a global?

Reply:
Good catch. This global was added in https://github.com/mozilla-mobile/fenix/pull/4088/files#diff-59f90d509bc6ae2c9d877250c5ba49c5R18, I thought it was used elsewhere, but it's not. I removed it.